ts/theme/components/Icon.ts

```diff
@@ -0,0 +1,9 @@
+import { Theme } from "../types";
```

Comment:
What is this useful for?

Reply:
I have delete de Icon component. It's not useful.